Libraries/Inspector/Inspector.js

```diff
@@ -17,17 +17,61 @@
 const Dimensions = require('Dimensions');
 const InspectorOverlay = require('InspectorOverlay');
 const InspectorPanel = require('InspectorPanel');
-const InspectorUtils = require('InspectorUtils');
 const Platform = require('Platform');
 const React = require('React');
+const ReactNative = require('ReactNative');
 const StyleSheet = require('StyleSheet');
 const Touchable = require('Touchable');
 const UIManager = require('UIManager');
 const View = require('View');
+const invariant = require('invariant');
 
-if (window.__REACT_DEVTOOLS_GLOBAL_HOOK__) {
-  // required for devtools to be able to edit react native styles
-  window.__REACT_DEVTOOLS_GLOBAL_HOOK__.resolveRNStyle = require('flattenStyle');
+export type ReactRenderer = {
+  // Fiber
+  findHostInstanceByFiber: (fiber: any) => ?number,
+  findFiberByHostInstance: (hostInstance: number) => any,
+
+  // Stack
+  ComponentTree: {
+    getNodeFromInstance: (instance: any) => ?number,
+    getClosestInstanceFromNode: (tag: number) => any,
+  },
+};
+
+function findRenderer(): ReactRenderer {
+  const hook = window.__REACT_DEVTOOLS_GLOBAL_HOOK__;
```

Comment:
Can we centralize the place we read this to a single place since it is kind of a hack? Even if it is a function call. Also, can we avoid `window`? It's lame that we have to add a global window object in RN.